test/benchmarks_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:convert' show JsonEncoder;
+import 'dart:io';
+
+import 'package:test/test.dart';
+
+import 'package:gallery/benchmarks/common.dart';
+
+import 'package:web_benchmarks/server.dart';
+
+final listOfBenchmarks = <String>[
+  galleryStudiesPerf,
+  galleryUnanimatedPerf,
+  galleryAnimatedPerf,
+  galleryScrollPerf,
+];
+
+Future<void> main() async {
+  test('Can run a web benchmark', () async {
+    print ('Starting web benchmark tests ...');
+
+    final taskResult = await serveWebBenchmark(
```

Comment:
Might be better to run `run_benchmarks.dart`'s `main` directly instead

Reply:
This test does not need to display the result, but `run_benchmarks.dart` does.